oem/supermicro/ikvm.go

```diff
@@ -0,0 +1,78 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package supermicro
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/stmcginnis/gofish/common"
+)
+
+type IKVMInterface string
+
+const (
+	JavaIKVMInterface IKVMInterface = "JAVA plug-in"
+	HTMLIKVMInterface IKVMInterface = "HTML 5"
+)
+
+type IKVM struct {
+	common.Entity
+
+	OdataType        string        `json:"@odata.type"`
```

Comment:
We've standardized on `OData` (versus `Odata`) elsewhere. Mind changing these to be consistent?

Reply:
I changed it.